generators/entity-client/templates/angular/src/main/webapp/app/entities/list/entity-management.component.ts.ejs

```diff
@@ -16,6 +16,12 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
+<%_
+  const notSortableFieldsAfterSearch = !searchEngine ? '' : fields
+      .filter(field => !field.hidden && !field.fieldTypeBoolean && !field.fieldTypeNumeric && !field.fieldTypeTemporal)
+      .map(field => `'${field.fieldName}'`)
+      .join(', ');
+_%>
```

Comment:
What about putting that in a constant in the generated source?
May be useful at other places.